api/src/champion-mastery/service.ts

```diff
@@ -1,3 +1,15 @@
-export function getChampionMastery() {
-  return { hello: 'world' };
+import { Constants } from 'twisted';
+import { riotClient } from '../riot-client';
+
+export async function getChampionMastery() {
+  const summoner = await riotClient.Summoner.getByName(
+    'fifa4life',
+    Constants.Regions.AMERICA_NORTH
+  );
+  const res = await riotClient.Champion.masteryBySummoner(
+    summoner.response.id,
+    Constants.Regions.AMERICA_NORTH
+  );
+
+  return res.response;
```

Comment:
This is semi-temporary code. Will be cleaned up in #7, and will get rid of hard coding